src/app/input-validator/input-validator.component.html

```diff
@@ -1,14 +1,16 @@
-@if (isInvalid) {
-  <div class="error-message">
-    @if (isInvalid) {
-      <p>
-        Field required.
-      </p>
-    }
-    @if (hasMinLengthError) {
-      <p>
-        Field must be at least {{ minLength }} characters long.
-      </p>
-    }
-  </div>
+@if (property !== null) {
+  @if (isInvalid) {
+    <div class="error-message">
+      @if (isInvalid) {
+        <p>
+          Field required.
+        </p>
+      }
+      @if (hasMinLengthError) {
```

Comment:
it would be nice to avoid calling methods in the template as that can impact performance, you can use the same approach you used previously